### ISSUE TITLE
Could not read file: [...] org.conf leads to crash

### ISSUE THREAD

**Issue by hvolkmer** [pki-io/core#91]
Running this command (outside of org directory)
`pki.io ca new tlabs-dev --dn-o falcon --dn-ou qa1 --tags consul`
leads to
```
Loading admin app
&{{{0xc20805c480 map[] false {0 0} [true true false false false true] 0xc20805c540 0} 0xc2080749c0 0xc20805b200}}
*************************************************
*                CONGRATULATIONS                *
*************************************************

You may have just found a bug in pki.io :)

Please let us know by raising an issue on GitHub here: https://github.com/pki-io/core/issues

Or by dropping an email to: dev@pki.io

If possible, please include this full error message, including the below panic,
and anything else relevant like what command you ran.

Many thanks,
The pki.io team

The error was: Couldn't read org config: Could not read file: open /Users/hvolkmer/Downloads/pki.io/org.conf: no such file or directory


panic: ...

goroutine 1 [running]:
main.checkAppFatal(0x4dad30, 0x1c, 0xc20802b030, 0x1, 0x1)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/helpers.go:46 +0x22c
main.(*AdminApp).LoadOrgConfig(0xc20805a7c0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/adminApp.go:233 +0x133
main.(*AdminApp).Load(0xc20805a7c0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/adminApp.go:256 +0x106
main.caNew(0xc20808a420, 0x0, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/runCA.go:25 +0x9b6
main.runCA(0xc20806e090, 0x9, 0x9, 0x0, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/runCA.go:185 +0x166
main.runCommand(0x7fff5fbff74f, 0x2, 0xc208060300, 0x8, 0x8, 0x0, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/pki.io.go:69 +0x321
main.main()
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/pki.io.go:52 +0x4b7

goroutine 5 [semacquire]:
sync.(*Cond).Wait(0xc20805a3c0)
	/usr/local/go/src/sync/cond.go:62 +0x9e
github.com/cihub/seelog.(*asyncLoopLogger).processItem(0xc20805c180, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:50 +0xc2
github.com/cihub/seelog.(*asyncLoopLogger).processQueue(0xc20805c180)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:63 +0x31
created by github.com/cihub/seelog.newAsyncLoopLogger
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:40 +0x8e

goroutine 6 [semacquire]:
sync.(*Cond).Wait(0xc20805a880)
	/usr/local/go/src/sync/cond.go:62 +0x9e
github.com/cihub/seelog.(*asyncLoopLogger).processItem(0xc20805c300, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:50 +0xc2
github.com/cihub/seelog.(*asyncLoopLogger).processQueue(0xc20805c300)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:63 +0x31
created by github.com/cihub/seelog.newAsyncLoopLogger
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:40 +0x8e

goroutine 7 [runnable]:
sync.(*Cond).Wait(0xc20805b200)
	/usr/local/go/src/sync/cond.go:62 +0x9e
github.com/cihub/seelog.(*asyncLoopLogger).processItem(0xc20805c4e0, 0x0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:50 +0xc2
github.com/cihub/seelog.(*asyncLoopLogger).processQueue(0xc20805c4e0)
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:63 +0x31
created by github.com/cihub/seelog.newAsyncLoopLogger
	/var/folders/n1/4ss_2rt10396zylvvdsjqv5cm4l758/T/tmp.qwP1TKnYGq/go/src/github.com/pki-io/admin/_vendor/src/github.com/cihub/seelog/behavior_asynclooplogger.go:40 +0x8e
```

Expected behavior in this case: Show error message about missing config file w/o crashing

I use this version: `pki.io 0.1.1-release1` on Mac OS X

**Comment by zeroXten** [comment]
Thanks for the bug report! I'll take a look and reproduce it. One thing I will say is that we've tried to take a defensive approach to errors, so things blow up at the hint of a problem rather than silently fail causing problems later on. However, as you've found, quite a few errors are classified as app errors when they should be user errors :)

**Comment by zeroXten** [comment]
Fixed in 0.1.2
